src/content/app/genome-browser/components/browser-cog/BrowserCogList.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import React, { useEffect, memo } from 'react';
+import React, { useEffect, memo, useRef } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
```

Comment:
We now need to use `useAppDispatch` and `useAppSelector` from `src/store` here and everywhere else